src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -100,10 +106,6 @@ object SnowflakeTelemetry {
       val stringWriter = new StringWriter
       throwable.get.printStackTrace(new PrintWriter(stringWriter))
       metric.put(TelemetryOOBFields.EXCEPTION_STACKTRACE, stringWriter.toString)
-    } else {
```

Comment:
To reviewers:
Removing this ELSE clause doesn't cause any actual change. Currently "throwable.isDefined" is true for all callers.

Reply:
let's not do this.

Reply:
I will update.